src/main/java/com/xiao/users/exception/GlobalExceptionHandler.java

```diff
@@ -29,17 +28,27 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
             String validationMsg = error.getDefaultMessage();
             validationErrors.put(fieldName, validationMsg);
         });
-        return new ResponseEntity<>(validationErrors, HttpStatus.BAD_REQUEST);
+
+        ErrorResponseDto errorResponseDTO = new ErrorResponseDto(
+            "Your Method Argument Is Not Valid",
```

Comment:
define a final static String

Refer: https://stackoverflow.com/questions/11677670/when-exactly-are-we-supposed-to-use-public-static-final-string